examples/seismic/source.py

```diff
@@ -73,17 +80,28 @@ def time_values(self):
 class PointSource(SparseTimeFunction):
     """Symbolic data object for a set of sparse point sources
 
-    :param name: Name of the symbol representing this source.
-    :param grid: :class:`Grid` object defining the computational domain.
-    :param time_range: :class:`TimeAxis` TimeAxis(start, step, num) object.
-    :param npoint: (Optional) number of sparse points represented by this source.
-    :param data: (Optional) data values to initialise point data.
-    :param coordinates: (Optional) point coordinates for this source.
-    :param space_order: (Optional) space discretization order.
-    :param time_order: (Optional) time discretization order (defaults to 2).
-    :param dtype: (Optional) data type of the buffered data.
-    :param dimension: (Optional) class:`Dimension` object for
-                      representing the number of points in this source.
+    Parameters
+    ----------
+    name : str
+        Name of the symbol representing this source.
+    grid : Grid
+        The computational domain.
+    time_range : TimeAxis
+        TimeAxis(start, step, num) object.
+    npoint : int, optional
+        Number of sparse points represented by this source.
+    data : ndarray, optional
+        Data values to initialise point data.
+    coordinates : ndarray, optional
+        Point coordinates for this source.
+    space_order : int or 3-tuple of ints, optional
```

Comment:
nitpicking: I think for PointSource this can/should only be ``int`` (not 3-tuple of ints)

Reply:
Fixed